sympy/ntheory/modular.py

```diff
@@ -253,3 +271,47 @@ def combine(c1, c2):
         if symmetric:
             return symmetric_residue(n, m), m
         return n, m
+
+
+def crt_cartesian(rem, mod):
+    """
+    Return a list of values whose elements give the remainders
+    in the Cartesian product of the supplied remainders when divided
+    by each of the supplied moduli. If the moduli are not co-prime,
+    you will receive a ValueError.
+
+    Parameters
+    ==========
+
+    rem: 2D list
+        Consists of list of lists whose cartesian product gives
```

Comment:
"List of lists, each containing the known remainders with respect to each modulus."